src/java/main/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -125,6 +125,8 @@
     private final ZooKeeperServerListener listener;
     private ZooKeeperServerShutdownHandler zkShutdownHandler;
 
+    private volatile boolean defaultTtlNodesEnabled = true;
```

Comment:
for the sake of clarity, why can't this be false?

Reply:
If it's false, then TTL nodes won't be available in Standalone mode.